tests/e2e/verify-install/capi/capi_test.go

```
@@ -0,0 +1,149 @@
+}
+
+func (c CapiDisabledModifierV1beta1) ModifyCRV1beta1(cr *v1beta1.Verrazzano) {
+	cr.Spec.Components.CAPI = &v1beta1.CAPIComponent{}
+	disabled := false
+	cr.Spec.Components.CAPI.Enabled = &disabled
+	t.Logs.Debugf("CapiDisabledModifierV1beta1 CR: %v", cr.Spec)
+}
+
+func (c CapiEnabledModifier) ModifyCR(cr *v1beta1.Verrazzano) {
+	if cr.Spec.Components.CAPI == nil {
+		cr.Spec.Components.CAPI = &v1beta1.CAPIComponent{}
+	}
+	enabled := true
+	cr.Spec.Components.CAPI.Enabled = &enabled
+	t.Logs.Debugf("CapiEnabledModifier CR: %v", cr.Spec)
+}
+
+func (c CapiEnabledModifierV1beta1) ModifyCRV1beta1(cr *v1beta1.Verrazzano) {
+	if cr.Spec.Components.CAPI == nil {
+		cr.Spec.Components.CAPI = &v1beta1.CAPIComponent{}
+	}
+	enabled := true
+	cr.Spec.Components.CAPI.Enabled = &enabled
+	t.Logs.Debugf("CapiEnabledModifierV1beta1 CR: %v", cr.Spec)
+}
+
+var t = framework.NewTestFramework("capi")
+
+var _ = t.AfterEach(func() {})
+
+var afterSuite = t.AfterSuiteFunc(func() {
+	m := CapiDisabledModifierV1beta1{}
+	var err error
+	if err != nil {
+		AbortSuite(fmt.Sprintf("Failed to get Verrazzano from the cluster: %v", err))
+	}
+
+	if isCapiInstalled {
+		update.UpdateCRV1beta1WithRetries(m, pollingInterval, waitTimeout)
+		update.ValidatePods(capiLabelValue, capiLabelKey, constants.VerrazzanoCapiNamespace, uint32(0), false)
+	}
+})
+
+var beforeSuite = t.BeforeSuiteFunc(func() {
+	m := CapiEnabledModifierV1beta1{}
+	var err error
+	kubeconfigPath := getKubeConfigOrAbort()
+
+	isCapiSupported, err = pkg.IsVerrazzanoMinVersion("1.6.0", kubeconfigPath)
+	if err != nil {
+		AbortSuite(fmt.Sprintf("Failed to check Verrazzano version 1.6.0: %v", err))
+	}
+	if isCapiSupported {
+		update.UpdateCRV1beta1WithRetries(m, pollingInterval, waitTimeout)
+		update.ValidatePods(capiLabelValue, capiLabelKey, constants.VerrazzanoCapiNamespace, uint32(4), false)
+	}
+	inClusterVZ, err = pkg.GetVerrazzanoInstallResourceInClusterV1beta1(kubeconfigPath)
```

Error is not handled

---

Fixed it, thanks.